include/cantera/kinetics/Kinetics.h

```diff
@@ -211,9 +223,32 @@ class Kinetics
      *
      * If a -1 is returned, then the phase is not defined in the Kinetics
      * object.
+     * @deprecated  To be removed after %Cantera 3.2. Use 2-parameter version instead.
      */
     size_t phaseIndex(const string& ph) const {
+        warn_deprecated("Kinetics::phaseIndex", "'raise' argument not specified; "
+            "Default behavior will change from returning -1 to throwing an exception "
+            "after Cantera 3.2.");
+        return phaseIndex(ph, false);
+    }
+
+    /**
+     * Return the phase index of a phase in the list of phases defined within
+     * the object.
```

Comment:
I know this is just copy/pasted from before, but this is a good opportunity for improvement. Perhaps something along the lines of:
```suggestion
     * Return the index of a phase among the phases participating in this kinetic mechanism.
```